config/_default/menus/main.en.yaml

```diff
@@ -1361,11 +1361,11 @@ menu:
       parent: dashboards_ddsql_editor_reference
       identifier: dashboards_ddsql_editor_reference_aggregation_functions
       weight: 11025
-    - name: Window Functions
-      url: dashboards/ddsql_editor/reference/window_functions
-      parent: dashboards_ddsql_editor_reference
-      identifier: dashboards_ddsql_editor_reference_window_functions
-      weight: 11026
+    # - name: Window Functions
+    #   url: dashboards/ddsql_editor/reference/window_functions
+    #   parent: dashboards_ddsql_editor_reference
+    #   identifier: dashboards_ddsql_editor_reference_window_functions
+    #   weight: 11026
```

Comment:
Commented out since this page will be in Private until it's tested to be stable.